scripts/lib/github-issues.mjs

```diff
@@ -0,0 +1,230 @@
+/**
+ * Note: This module is only here to avoid including octokit.js with hundreds of dependencies,
+ * which would slow down the install task of our GitHub workflows.
+ * 
+ * If we want to switch to octokit.js at one point, we should make this code independent from
+ * our scripts by creating separate GitHub actions with their own repos and compilation steps.
+ */
+
+import fetch from 'node-fetch';
```

Comment:
Wondering if we could run this on Node 18 and use the new `fetch` global? This is fine too though.